src/Dependencies/Installer.php

```diff
@@ -27,6 +29,17 @@ public function install()
                 if ($result->isErr()) {
                     return $result;
                 }
+                $result = $this->inspectionChecker->check($type, $dep);
+                if (!$result->isErr()) {
+                    $inspectionDetails = $result->unwrap();
+                    if (!empty($inspectionDetails)) {
+                        echo sprintf("%s\n", $inspectionDetails);
+                    }
+                } else {
+                    $error = $result->getErr();
+                    echo sprintf("[ERROR] %s\n", $error);
+                }
+                echo "\n";
```

Comment:
I wonder if it'd be worth extracting the `inspectionChecker` stuff (lines 32-42) to a separate method within `Installer`? Then you could just call that from within `install()`, by doing something like `$this->displayInspections($type, dep);`. It'd make it easier to see at a glance what the `install()` method does.